models/issue.go

```diff
@@ -984,7 +984,6 @@ type NewIssueOptions struct {
 	Repo        *Repository
 	Issue       *Issue
 	LabelIDs    []int64
-	AssigneeIDs []int64
```

Comment:
Why removed this?

Reply:
I moved adding assignees outside the creation of new issues. Reason is that each added assignee creates a new comment (issue even) which is used in the notification.

Reply:
I think adding assignee comment should be in the transaction but not in the notifications?

Reply:
I am not sure I understand now. The added assignee comment is used when sending the notification (the comment hash is included in the mail).

Reply:
@lunny If I've read that right, it's not a comment in the sense of a row in the `comment` table but a message for the contents of the e-mail. @davidsvantesson, perhaps it should be renamed to `message` to avoid confusion with the existing model/structure?

Reply:
I am not sure if I still misunderstand you. Adding an assignee creates a Comment in the database of type CommentTypeAssignees. CommentTypeComment is another type. Maybe IssueEvent would have been a better name to reflect it is not only comments. Anyhow the Comment (of type Assignees) is used when constructing the e-mail.

Reply:
@davidsvantesson OK, sorry, I got confused about that. I understand @lunny's concern then. However, I'm not sure I'd block the creation of the issue because of an error while doing the assignments. That's why I think the error Flash is useful.

Reply:
The input data is validated before issue creation, but it is of course possible it can still fail because of unconsidered reasons / bugs. I think it is a good idea to use error flash.

Reply:
I've added an error flash now specifically for the case if add assignees fail when creating new issues/pr. Other error flashes I think can be for another PR. 
I am not sure how to handle it for API creation. Right now it can potentially give error although the issue itself is created.